test/views/pages/create_account/create_account_test.dart

```diff
@@ -53,7 +53,7 @@ void main() {
       ("heLlo ", space),
       ("A", short),
       ("a%", short),
-      ("0" * 21, long),
+      ("0" * 17, long),
```

Comment:
You created the _minStringLength and _maxStringLength variables in [lib/viewmodels/create_account_view_model.dart](https://github.com/ProximaEPFL/proxima/pull/330/files#diff-97ada99179c9b08c7aad6f9b10bb328c8b43fba692d917e3f167455dcb886e42). Do you think it would be a good idea to do the same thing in this test file? It would have the same purpose (avoid hardcoded values)

Reply:
Yes, super good idea! I actually struggled updating the tests because of the magic numbers so it would fix the problem.

Reply:
fixed in 2612b02017b1ecfdcd5fcf953ac01aa6eace7008